Hello, traveler. This is the beginning of a log file called: "Fascist Tweet Archiver".
Some basic system info about the host machine: posix.uname_result(sysname='Linux', nodename='linawesome', release='4.10.0-38-generic', version='#42~16.04.1-Ubuntu SMP Tue Oct 10 16:32:20 UTC 2017', machine='x86_64')
This log was begun Tuesday, 25 December 2018 at 23:07



INFO: read target account data; we're tracking 65 accounts
WARNING: no ProtocolError (got exception "name 'ProtocolError' is not defined"); trying requests.packages.urllib3.exceptions instead
NOTE: successfully imported from requests
